docs/siem/detections/detections-ui-exceptions.asciidoc

```diff
@@ -0,0 +1,159 @@
+[[detections-ui-exceptions]]
+[role="xpack"]
+== Rule exceptions and value lists
+
+To prevent the creation of unwanted alerts, you can add exceptions to detection
+and Endpoint rules. Exceptions contain the source event conditions that
+determine when alerts are not generated. They provide a convenient way of
+allowing trusted processes and network activity to function without producing
+unnecessary noise.
+
+You can add multiple exceptions to one rule.
+
+IMPORTANT: When you add an exception to the
+<<endpoint-rule-exceptions, Elastic Endpoint Security>> rule, you can select to
+add the exception to the Endpoint. When selected, the exception is added to
+both the detection rule *and* the Elastic Endpoint agent on your hosts.
+
+In addition to defining exception queries for source event values, detection
+rule exceptions can be used with value lists. Value lists are lists of items
+with the same {es} {ref}/mapping-types.html[data type]. You can create value
+lists with these types:
+
+* `keyword` (many {ecs-ref}/ecs-field-reference.html[ECS fields] are keywords)
+* `ip`
+* `ip_range`
+* `text`
+
+After creating value lists, you can use `is in list` and `is not in list`
+operators to define exceptions.
+
+[float]
+=== Create a value list
+
+To create a value list for use with exceptions:
+
+. Prepare a `txt` or `csv` file with all the values you want to use for
+determining exceptions from a single list. If you use a `txt` file, newlines
+act as value delimiters.
++
+NOTE: All values in the file must be of the same {es} type.
+
+. Go to *Security* -> *Detections* -> *Manage detection rules*.
+. Click *Upload value lists*.
++
+The *Upload value lists* window opens.
+
+[role="screenshot"]
+image::images/upload-lists-ui.png[]
+
+. Select the list type (`Keywords`, `IP addresses`, `IP ranges`, or
+`Text`)
+. Drag or select the `csv` or `txt` file that contains the values.
+. Click *Upload list*.
+
+[float]
+[[detection-rule-exceptions]]
+=== Add detection exceptions to a rule
+
+You can add exceptions to a rule via the Rule details page or the Alerts table.
+When you add an exception via the Alerts table, you can close either the alert
+via which you are adding the exception or all alerts that meet the exception's
+criteria. When you add an exception via the Rule details page, you can close
+all alerts that meet the exception's criteria.
```

Comment:
Nit: this reads strange to me. I feel that the checkboxes themselves are self-explanatory, so maybe we just want to generally call out that functionality? Something like "When adding an exception you also have the opportunity to immediately close alerts matching its criteria."